apps/clkinfostopw/clkinfo.js

```diff
@@ -74,4 +74,4 @@
             }
         ]
     };
-});
+}); // FIXME: semi-colon added automatically when Typescript generates Javascript file?
```

Comment:
@bobrippling Since you know TS, can the generated JS be made to not add a semi-colon here?

Reply:
Good question - no not via ts itself but we could add a post-processing step. That would also potentially help with the variable names, [such as `_1`](https://github.com/espruino/BangleApps/pull/3565#issuecomment-2342930224).

If that sounds good to you, I'll pop something onto the end of this PR

Reply:
Super - please do!

Reply:
This is ready now. So when you get the time please add on the post-processing step 🙏

Reply:
Sorted - see 3e67088354e0 for the workings 🔍